ggshield/core/oauth.py

```diff
@@ -333,6 +333,18 @@ def _check_existing_token(self) -> bool:
         click.echo(message)
         return True
 
+    def get_server_error(self, error_code: str) -> str:
+        """
+        Return the human readable message associated to the given error code
+        """
+        if error_code == "too_many_tokens":
+            url = urljoin(self.dashboard_url, "/api/personal-access-tokens")
+            return (
+                "Maximum number of personal access tokens reached. Could not provision a new personal access token.\n"
+                f"Go to your workspace to manage your tokens: {url}"
+            )
+        return "An unknown server error occured."
```

Comment:
```suggestion
        return f"An unknown server error has occurred (error code: {error_code})."
```
(fixes a typo, and adds the value of `error_code`, useful for debugging)